README.md

```diff
@@ -2,21 +2,122 @@
 
 # PMIx Standard
 
-This repository contains the LaTeX source for the PMIx standard document, and discussions regarding the PMIx standard.
+This repository contains the LaTeX source for the PMIx Standard document, and discussions regarding the PMIx standard.
 
-## Contributing to the PMIx Standard
+Contributions of any form (e.g., use cases, questions, suggestions) are encouraged from anyone interested in the evolution of the PMIx Standard.
+You can participate in the discussion on the [PMIx Forum mailing list](https://groups.google.com/group/pmix-forum),
```

Comment:
Minor nit: somewhere I think we decided to only (mostly?) use the mailing list for announcements/broadcasts rather than discussions (which should mostly happen on GitHub via PRs and issues).  Following that should we emphasize PRs/Issues by putting them first in the list?

Reply:
Yeah, we can reword a bit to better emphasize that. The next paragraph really goes into more detail on it. Folks may still post to the mailing list with general procedural questions ("how do I file a clarification?") and we can redirect them to the GitHub site.

Maybe rephrase like:
> You can participate in the discussion on the Pull Requests/Issues on this GitHub repository or on the regular teleconference (announced on the [PMIx Forum mailing list](https://groups.google.com/group/pmix-forum)). The PMIx Forum mailing list is primarily used for announcements and general questions about the PMIx Standard process (e.g., "When is the next release scheduled?" or "I am having trouble filing an issue").

Reply:
I just committed the new language. Thanks.